android/gltfio-android/src/main/java/com/google/android/filament/gltfio/FilamentAsset.java

```diff
@@ -21,6 +21,22 @@
 import com.google.android.filament.Box;
 import com.google.android.filament.Entity;
 
+/**
+ * Owns a bundle of Filament objects that have been created by AssetLoader.
+ *
+ * For usage instructions, see the documentation for AssetLoader.
+ *
+ * This class owns a hierarchy of entities that have been loaded from a glTF asset. Every entity has
+ * a TransformManager component, and some entities also have a NameComponentManager and/or
+ * RenderableManager components.
+ *
+ * In addition to the aforementioned entities, an asset has strong ownership over a list of
+ * VertexBuffer, IndexBuffer, MaterialInstance, Texture, and, optionally, a simple animation engine
+ * (Animator).
+ *
+ * Clients can ResourceLoader to create Texture objects, compute tangent quaternions, and upload
+ * data into vertex buffers and index buffers.
+ */
```

Comment:
It is good practice to use @see at the end to point to related classes, APIs, etc.

```
/**
 * This class uses a {@link Foo} to do bar.
 * @see Foo
 * @see Foo#bar()
 */
```